subsys/debug/ppi_trace/ppi_trace.c

```diff
@@ -196,6 +196,35 @@ void *ppi_trace_pair_config(uint32_t pin, uint32_t start_evt, uint32_t stop_evt)
 #endif
 }
 
+bool ppi_trace_dppi_config(uint32_t pin, uint32_t dppi_ch)
+{
+#ifdef DPPI_PRESENT
+	uint32_t task;
+	int gpiote_ch;
+	nrf_gpiote_task_t task_id;
+
+	gpiote_ch = gpiote_channel_alloc(pin);
+	if (gpiote_ch < 0) {
+		LOG_ERR("Failed to allocate GPIOTE channel.");
+		return false;
```

Comment:
It would be better to use error codes `-ENOMEM` here and `-ENOTSUP` if DPPI is not present.

Reply:
done